service/distros.go

```diff
@@ -47,6 +47,14 @@ func (uis *UIServer) distrosPage(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
+	var createDistro bool
+	resourcePermissions, ok := permissions[evergreen.SuperUserPermissionsID]
```

Comment:
Admin permissions should be a separate permission type, otherwise there won't be any scope to match on